pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -943,7 +943,7 @@ func handleCreate(ctx *domainContext, key string, config *types.DomainConfig) {
 	// Do we need to copy any rw files? !Preserve ones are copied upon
 	// activation.
 	for _, ds := range status.DiskStatusList {
-		if status.IsContainer {
+		if zconfig.Format_CONTAINER == ds.Format {
```

Comment:
Nit.. from a readability POV.. its better to reverse the order of comparision:

if ds.Format == zconfig.Format_CONTAINER {

This is much more readable and understandable..

Reply:
Resolved.